src/mon/OSDMonitor.cc

```diff
@@ -559,6 +559,7 @@ void OSDMonitor::on_active()
       op->mark_osdmon_event(__func__);
       dispatch(op);
       ls.pop_front();
+      // no need to mon->no_reply since this is a new quorum
```

Comment:
i think the reason we don't need to `mon->no_reply()` here is that the `op` is not handled yet. they will be taken care of by `PaxosService::dispatch()`.  we will send `no_reply()` when actually processing them.

Reply:
oh right, i didn't see the dispatch() call there.   removing this!